### ISSUE TITLE
Player Health bar does not work, but instead for some minions it does

### ISSUE THREAD

**Issue by danthespal** [zaafar/GameOverlay#219]
v0.344, for me is an old issue I used health bar from the game till now when I want to test it in the overlay but with it, did not work for a long time.

![image](https://user-images.githubusercontent.com/55256796/168990812-3a69b866-de61-4983-908a-99d0b0025517.jpg)

**Comment by zaafar** [comment]
The entities on which HB doesn’t work, share their component data.

**Comment by danthespal** [comment]
![image](https://user-images.githubusercontent.com/55256796/169067122-f0497d2e-2e53-455a-bab8-9e10351f3693.png)

this one?

**Comment by zaafar** [comment]
no, in your screenshot do you see "Awake Entities" (last line).
same thing you are showing in Player, show in that sub-section.


NOTE: Select the entity who's health bar isn't showing, once you open Awake Entities sub-section, you will figure out how to select entity. it's very intuitive.

**Comment by danthespal** [comment]
![image](https://user-images.githubusercontent.com/55256796/169528068-4b279fd7-a20e-45c0-8883-0b3311d9b163.png)

**Comment by Treasure-Box** [comment]
I recently started checking out a minion build and noticed that HP bars for players and minions were not working correctly, so i cam to check to see if it was already a reported issue. I would also support the return of these HP bars as both size and color customize help a lot as the stock game once are just a bit to small/hard to see.

**Comment by Treasure-Box** [comment]
@zaafar check above comment.

**Comment by zaafar** [comment]
Will add it back then.

**Comment by Treasure-Box** [comment]
Oh yeah then that would be another issue as friendly/player minions do not currently have a HP bar rendered for them through GH regardless of option toggle.

**Comment by danthespal** [comment]
because I test it every version and did not work, so I used the in game HP bar, but sure HP bar in the overlay would be nice, but I don't know how will work with minions, maybe a box to disable or enable them.
